### PR TITLE
Fix duplicate imported files

### PR DESCRIPTION
**Pull request by nosami**

### CODE REVIEW

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/ProjectTests.fs [hunk 1]
```diff
@@ -125,6 +173,245 @@ type ProjectTests() =
...
+        //            |> List.filter(fun i -> i.StartsWith("Assets.xcassets\\AppIcon.appiconset"))
+        //            |> List.sort
+
+        //        let expected =
+        //            ["Assets.xcassets\\AppIcon.appiconset\\Icon-App-29x29%401x.png"
+        //             "Assets.xcassets\\AppIcon.appiconset\\Icon-App-76x76%402x.png"
+        //             "Assets.xcassets\\AppIcon.appiconset\\Icon-App-test1.png"
+        //             "Assets.xcassets\\AppIcon.appiconset\\Icon-App-test2.png"]
+        //        Assert.AreEqual(expected, includes, sprintf "%A" includes)
+        //}
```
Comment (mrward) [inline main/external/fsharpbinding/MonoDevelop.FSharp.Tests/ProjectTests.fs]: I guess this commented out code can be removed.

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs [hunk 2]
```diff
@@ -193,6 +221,3 @@ type ``Template tests``() =
     [<Test;AsyncStateMachine(typeof<Task>)>]member x.``FSharpGtkProject``()= test "FSharpGtkProject"
     [<Test;AsyncStateMachine(typeof<Task>)>]member x.``MonoDevelop FSharp LibraryProject``()= test "MonoDevelop.FSharp.LibraryProject"
     [<Test;AsyncStateMachine(typeof<Task>)>]member x.``FSharpNUnitLibraryProject``()= test "FSharpNUnitLibraryProject"
-    [<Test;AsyncStateMachine(typeof<Task>)>]
-    member x.``Xamarin Forms FSharp FormsApp Shared``() =
-        testWithParameters ("Xamarin.Forms.FSharp.FormsApp", "CreateSharedAssetsProject=True;CreatePortableDotNetProject=False")
```
Comment (mrward) [inline main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs]: Was this test supposed to be removed? I think that is the one I added to test shared asset projects.

File: main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs [hunk 3]
```diff
@@ -891,6 +891,33 @@ public class MSBuildGlobTests: TestBase
 			}
 		}
 
+		[Test]
+		public async Task FilesImportedAreMarkedAsImported ()
+        {
```
Comment (mrward) [inline main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs]: Formatting here is a bit wonky.

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs (hunk 2 above)
Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs]: Oops..No

File: main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs (hunk 3 above)
Comment (nosami) [reply main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs]: Emacs merge :) Will fix

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/ProjectTests.fs (hunk 1 above)
Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Tests/ProjectTests.fs]: fixed

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs (hunk 2 above)
Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Tests/TemplateTests.fs]: fixed

File: main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs (hunk 3 above)
Comment (nosami) [reply main/tests/MonoDevelop.Core.Tests/MonoDevelop.Projects/MSBuildGlobTests.cs]: fixed

**Pull request merged**